### PR TITLE
UI static files for modules re-organized

### PR DESCRIPTION
**Pull request by smpadhy**
The static resources for  different modules are re-organized. For example, all html files for NDA module are moved from eapp/public/html to eapp/public/nda/html/.
So, now any new static file (html/js/css) for a module will go to folder  eapp/public/MODULE-NAME/<html|css|js>/filename.<html|css|js>

### CODE REVIEW

File: eapp/app.js [hunk 1]
```diff
@@ -135,13 +135,7 @@ module.exports = () => {
...
   });
 
-  app.use(express.static(path.join(__dirname, 'public/css')))
-  app.use(express.static(path.join(__dirname, 'public/html')))
-  app.use(express.static(path.join(__dirname, 'public/js')))
-  app.use(express.static(path.join(__dirname, 'public/terms')))
-  app.use(express.static(path.join(__dirname, 'public/lib')))
-  app.use(express.static(path.join(__dirname, 'public/images')))
-
+  app.use(express.static(path.join(__dirname, 'public/')))
```
Comment (satra) [inline eapp/app.js:138]: is `public` necessary or could we use something like: `app`

**Review by satra — APPROVED** [review]
other than the public vs app question, the general reorganization looks good.

File: eapp/app.js (hunk 1 above)
Comment (smpadhy) [reply eapp/app.js:138]: No. It is in the general structure of an express app where static resources are put in public directory. I think we can rename 'public'. But I would like to keep the 'public' folder  for common static files (js/css/html) for the app.
We can create another folder outside 'public', may be 'modules' with all the modules UI statics file, that is moving modules from 'public' folder to 'modules' folder.

Currently, under brainverse repo, we have 'eapp' directory with all codes in there. Using 'app' for 'public' seems duplicating. 

here is the scheme if we replace 'public' with 'app'
S1:
eapp/
&nbsp; &nbsp; config/
&nbsp; &nbsp; app/
 &nbsp; &nbsp; &nbsp;   css/
   &nbsp; &nbsp; &nbsp;   html/
    &nbsp; &nbsp; &nbsp;  js/    
&nbsp; &nbsp; &nbsp; nda/
     &nbsp; &nbsp; &nbsp; project-planner/
  &nbsp; &nbsp; routes/
  &nbsp; &nbsp; utils/

S2: if we keep 'public' and create a new folder 'modules'
eapp/
  &nbsp; &nbsp; config/
  &nbsp; &nbsp;public/
  &nbsp; &nbsp; &nbsp;   css/
  &nbsp; &nbsp; &nbsp;    html/
   &nbsp; &nbsp; &nbsp;   js/
   &nbsp; &nbsp;modules/
    &nbsp; &nbsp; &nbsp; nda/
    &nbsp; &nbsp; &nbsp; project-planner/
 &nbsp; &nbsp; routes/
&nbsp; &nbsp;  utils/


@satra Which one do you prefer?

Comment (satra) [reply eapp/app.js:138]: and this cannot be changed i presume?

could we change the route for each static app through express, so we avoid `/public`?

Comment (smpadhy) [reply eapp/app.js:138]: @satra Sorry, for the confusion. Please see my updated detailed answer above. 
Also note that /public will not be there in the routes. For example, if you are refereing to a nda module html file, then you can access by 127.0.0.1/nda/html/x.html. This is also true if we make 'modules' folder static resources inside express routes.

Comment (smpadhy) [reply eapp/app.js:138]: Reorganized the files to based on S2 structure (as mentioned above).

**Pull request merged**